### ISSUE TITLE
Wrong rel=canonical 

### ISSUE THREAD

**Issue by Lofesa** [ahmedkaludi/accelerated-mobile-pages#458]
In desing-X index.php you have 
`$current_archive_url = home_url( $wp->request );
$amp_url = trailingslashit($current_archive_url);`

but when you are in amp page $wp-request has the triling /amp, so you got a wrong rel=canonical

**Comment by khaledMohammed000** [comment]
Hey @Lofesa 

am sorry for the late reply, I cant recreate the issue,  screenshot would really help and also URL

**Comment by ahmedkaludi** [comment]
This was fixed in 0.9.40 Update. Thank you so much @Lofesa  for reporting this. Next time, I will keep a better eye on tickets and will act on it faster.

**Comment by Lofesa** [comment]
Hi @ahmedkaludi 
Seems is here again.
In templates/design-manager/design-2/footer.php you get this code:
```
  if( is_archive() ) {
    global $wp;
    $ampforwp_backto_nonamp = esc_url( untrailingslashit(home_url( **$wp->request** )).'?nonamp=1' );
  }
```
whych get this on archive pages

`https://spublicos.intersindicalrm.org/category/carm/educacion/amp/?nonamp=1`

and don´t go to the non-amp version
